introduction.md

```diff
@@ -2,4 +2,6 @@ I am a developer who uses Git everyday. It is nice to know more about it.
 My favorite course is javaScript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?